vllm/worker/model_runner_base.py

```diff
@@ -262,6 +262,17 @@ def __init__(
     def __getattr__(self, attr):
         return getattr(self.model_runner, attr)
 
+    def __setattr__(self, name, value):
+        """
+         Ensure that setting the 'model_runner' attribute
+         does not delegate to model_runner
+         """
+
+        if name == "model_runner":
+            object.__setattr__(self, name, value)
+        else:
+            setattr(self.model_runner, name, value)
+
```

Comment:
necessary change: in spec_decode_worker, it will update model_runner.return_hidden_states property, without setattr, we can't actually set the property in hpu_model_runner.
Which is the root cause for None type issue in error log